README.md

```diff
@@ -118,6 +118,24 @@ helm install capi-operator capi-operator/cluster-api-operator --create-namespace
 ```
 This will take some time to install and can be verified it is complete by seeing two ready pods in the `azure-infrastructure-system` namespace. 
 
+In order to install CAPI Operator with additional CRDs, you will need to use `helm install` with a `values.yaml` file and customize the `azureserviceoperator-controller-manager` deployment like so:
+
+```yaml
+additionalDeployments:
+  azureserviceoperator-controller-manager:
+    deployment:
+      containers:
+      - name: manager
+        args:
+          --crd-pattern: "documentdb.azure.com/*;managedidentity.azure.com/*"
+```
+
+To use the example `values.yaml` file, run the following command:
+```shell
+helm install capi-operator capi-operator/cluster-api-operator --create-namespace -n capi-operator-system --wait --timeout 90s -f gitops/environments/default/addons/cluster-api-provider-azure/values.yaml
```

Comment:
Please replace the instructions (lines 111-117) with this new command as the default.